cmd/gardenlet/app/gardenlet.go

```diff
@@ -339,8 +343,19 @@ func NewGardenlet(ctx context.Context, cfg *config.GardenletConfiguration) (*Gar
 		return nil, err
 	}
 
+	gardenClusterIdentity := &corev1.ConfigMap{}
+	if err := k8sGardenClient.Client().Get(ctx, kutil.Key(metav1.NamespaceSystem, v1beta1constants.ClusterIdentity), gardenClusterIdentity); err != nil {
+		return nil, err
+	}
+
+	clusterIdentity, ok := gardenClusterIdentity.Data[v1beta1constants.ClusterIdentity]
+	if !ok {
+		return nil, fmt.Errorf("unable to get Gardener`s cluster identity: %v", err)
```

Comment:
```suggestion
		return nil, errors.New("unable to get Gardener`s cluster identity")
```
It makes no sense to wrap this error here, if you want it wrapped you'd better do it where it actually occurred above.

Reply:
Done.